src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,57 @@
+// Can't think of a better way to do this
+// The initial shapeshift form is 311px tall
+// the .shapeshift__initial-wrapper class is only added when the form is being loaded
+// Once the form is rendered, there is a very smooth transition because the height doesn't change
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 311px;
+}
+
+.shapeshift__content {
+  .spinner {
+    margin-top: $spacing-vertical * 3;
+  }
+}
+
+.shapeshift__success {
+  color: var(--color-primary);
+}
+
+.shapeshift__error,
+.shapeshift__maxdeposit {
```

Comment:
This CSS should be able to go.

Reply:
I'll keep the min-height on `shapeshift__maxdeposit` to avoid content jumping when we fetch new max limits after the user changes the origin coin